src/runtime/components/forms/FormGroup.ts

```diff
@@ -15,6 +15,13 @@ export default defineComponent({
       type: String,
       default: null
     },
+    size: {
+      type: String,
+      default: () => appConfig.ui.button.default.size,
```

Comment:
I think it would be better to use the default for `input`
`appConfig.ui.input.default.size`

Reply:
@henrycunh Nice work Good luck